pkg/metrics/metrics.go

```diff
@@ -31,10 +31,17 @@ const (
 	backupAttemptCount          = "backup_attempt_total"
 	backupSuccessCount          = "backup_success_total"
 	backupFailureCount          = "backup_failure_total"
+	backupSecondsCount          = "backup_seconds_total"
 
 	scheduleLabel = "schedule"
 )
 
+// SecondsInMinute returns the number of seconds in a minute.
+// This is mostly a helper to create Prometheus histogram buckets cleanly.
+func SecondsInMinute(minutes uint64) float64 {
```

Comment:
Not sure you need this method. Suggest creating a const like `secondsInMinute = 60` and 
```
Buckets: []float64{
	SecondsInMinute(1 * secondsInMinute),
	SecondsInMinute(2 * secondsInMinute),
	SecondsInMinute(3 * secondsInMinute),
	SecondsInMinute(4 * secondsInMinute),
	SecondsInMinute(5 * secondsInMinute),
	SecondsInMinute(6 * secondsInMinute),
	SecondsInMinute(7 * secondsInMinute),
	SecondsInMinute(8 * secondsInMinute),
	SecondsInMinute(9 * secondsInMinute),
	SecondsInMinute(10 * secondsInMinute),
},
```
WDYT?

Reply:
Good suggestion, I'll go with a `float64` constant and use it directly, without a function call.